models/timm/beit_base_patch16_224.py

```diff
@@ -0,0 +1,19 @@
+# labels: name::beit_base_patch16_224 author::timm
```

Comment:
Any other labels available? Like the CV task (classification/segmentations/detection)?

Reply:
Or at least putting "CV" on all them so they can be filtered vs. NLP

Reply:
Added CV as the task of those models